config/openshift/manifests/addon-operator.csv.yaml

```diff
@@ -101,6 +101,7 @@ spec:
           - operatorgroups
           - catalogsources
           - subscriptions
+          - installplans
```

Comment:
There's a section below that has fewer permissions for installplans, should we put this there? I don't see any big issue but with respect to the requirements we are fulfilling we only need to list/watch IPs.

Reply:
Thanks, it's a better approach to always have CRUD permissions over our operator resources (the ones we manage) as we're the only & primary orchestrators of all the addons. While I understand your point, I don't see any reason for this to be considered too permissive.